python/ray/autoscaler/_private/kuberay/run_autoscaler.py

```diff
@@ -51,17 +63,29 @@ def run_kuberay_autoscaler(cluster_name: str, cluster_namespace: str):
         cluster_name, cluster_namespace
     )
 
-    Monitor(
-        address=ray_address,
-        # The `autoscaling_config` arg can be a dict or a `Callable: () -> dict`.
-        # In this case, it's a callable.
-        autoscaling_config=autoscaling_config_producer,
-        monitor_ip=head_ip,
-        # Let the autoscaler process exit after it hits 5 exceptions.
-        # (See ray.autoscaler._private.constants.AUTOSCALER_MAX_NUM_FAILURES.)
-        # Kubernetes will then restart the autoscaler container.
-        retry_on_failure=False,
-    ).run()
+    # Init GCS client
+    _initialize_internal_kv(GcsClient(ray_address))
+    if is_autoscaler_v2(fetch_from_server=True):
```

Comment:
Where is the function? The function [is_autoscaler_v2](https://sourcegraph.com/github.com/ray-project/ray@6360a0673294c2d99309157dc06dd52ff1e8aedf/-/blob/python/ray/autoscaler/v2/utils.py?L759) here doesn't have an argument `fetch_from_server`.

Reply:
ah - this should be in a previous PR: https://github.com/ray-project/ray/pull/43355/files#diff-158db38f6f090af20c996794effb64a4dcf85d696000f3a2dd2bcb767326f401L788

sorry, the stacked PR isn't working that well. Let me see if i can get a cleaner stack.